public/js/main.js

```diff
@@ -94,24 +106,78 @@ $(document).on('ready', function() {
     });
   }
 
+  function updateMemberMap() {
+    $('#map_container').empty();
+    Papa.parse('/data/members_by_country.csv', {
+      download: true,
+      complete: function(results) {
+        var rawData = results.data;
+        var data = {};
+        for(var index = 1; index <= rawData.length; index++) { //Skip the first row
+          if(rawData[index] == undefined || rawData[index][0] == ""){
+            continue;
+          }
+          var fill = "veryLight";
+          var totalMembers = parseInt(rawData[index][1].split(',').join(' ').replace(/ /g,''));
+          if(totalMembers == 0) {
+            continue;
+          }
+          if(totalMembers >= 10000) {
+            fill = "veryHeavy";
+          }
+          else if(totalMembers >= 5000) {
+            fill = "heavy";
+          }
+          else if(totalMembers >= 1000) {
+            fill = "medium";
+          }
+          else if(totalMembers >= 500) {
```

Comment:
Hm, since this is looking at a range of values the only way I know how to switch this would be the following:

```
switch (true)
{
    case (totalMembers > 1000):
      stuff();
      break;

   case (totalMembers <= 1000):
      stuff()
      break; 
}
```

Feels sort of hacky because it defeats the point of using a switch, should I use it anyway?